sdk/core/azure-core/vcpkg.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "azure-core-cpp",
+    "version-string": "1.0.0",
+    "dependencies": [
+        "curl",
+        "openssl"
```

Comment:
Our vcpkg manifest template files (the ones in `./vcpkg/`, do have platform-specific dependencies, for example, Core does not have the openssl dependency on Windows.
```
    {
      "name": "openssl",
      "platform": "!windows & !uwp"
    },
```

Reply:
used it